packages/ubuntu_desktop_installer/lib/pages/installation_slides/installation_slides_page.dart

```diff
@@ -160,9 +160,8 @@ class _InstallationSlidesPageState extends State<InstallationSlidesPage> {
                   ],
                 ),
                 const SizedBox(height: kContentSpacing),
-                LinearProgressIndicator(
-                  value: model.isInstalling ? null : 0,
-                  backgroundColor: Theme.of(context).primaryColor.withAlpha(62),
+                const YaruLinearProgressIndicator(
+                  value: .5,
```

Comment:
please revert the test value :)

Reply:
Sorry, I'm doing a lot of mistakes these times 😕